variables.tf

```
@@ -10,6 +10,12 @@ variable "additional_tags" {
   default     = []
 }
 
+variable "alb_resource_label" {
+  description = "Enter the ALB and Target group in this format : app/load-balancer-name/load-balancer-id/targetgroup/target-group-name/target-group-id"
+  type        = string
+  default     = ""
```

Based on other review comments, I would suggest this default be updated to null, for better support of a unified target tracking policy resource.
```suggestion
  default     = null
```

Might also be worth thinking on we could dynamically look this information up instead of requiring it explicitly.  I would think if it is being used, we would also be getting the information to create the target group associations.  Nothing to account for now, but might be a good future improvement.

---

Wow that null is very helpful. Thanks for this suggestion. I have used this and it shortened the whole code and no need to think of using for_loop or dynamic.  I took the suggestion and used it and updated the new commit.